deparse.c

```diff
@@ -2366,8 +2403,27 @@ sqlite_deparse_direct_update_sql(StringInfo buf, PlannerInfo *root,
 		first = false;
 
 		sqlite_deparse_column_ref(buf, rtindex, attnum, root, false, true);
+
+		/* Get RangeTblEntry from array in PlannerInfo. */
+		rte = planner_rt_fetch(rtindex, root);
+		pg_attyp = get_atttype(rte->relid, attnum);
+		preferred_affinity = preferred_sqlite_affinity(rte->relid, attnum);
+
 		appendStringInfoString(buf, " = ");
+
+		special_affinity = (pg_attyp == UUIDOID && preferred_affinity == SQLITE3_TEXT) ||
+						   (pg_attyp == TIMESTAMPOID && preferred_affinity == SQLITE_INTEGER);
+		if (special_affinity)
+		{
+			elog(DEBUG3, "sqlite_fdw : aff %d\n", preferred_affinity);
+			if (pg_attyp == UUIDOID && preferred_affinity == SQLITE3_TEXT)
+				appendStringInfo(buf, "sqlite_fdw_uuid_str(");
+			if (pg_attyp == TIMESTAMPOID && preferred_affinity == SQLITE_INTEGER)
+				appendStringInfo(buf, "strftime(");
```

Comment:
I don't see any test case for this. Could you add it?

Reply:
There is such TCs, @son-phamngoc 
https://github.com/mkgrgis/sqlite_fdw/blob/8cf4145ef00cb0d344a5a63b2ffa2528ea7d9120/sql/16.0/extra/uuid.sql#L128C1-L138C1

For `timestamp` this fixes untested error and I'll plan special PR about `timestamp` mixed affinity data type. So TC about `timestamp` to extend existed TCs will be in this PR.

Reply:
I saw your test cases about UUID, but I did not see test cases about timestamp, so I added this comment.  
Do you mean you are going to add those test cases in another PR?  
If so, I think it is better to remove the source code about timestamp from this PR.  
You can add it again in your PR about timestamp. It will be more consistent.  
In this PR, the source code about timestamp has no test to verify, so we cannot allow to merge it. Please consider.

Reply:
@son-phamngoc , I have changed some lines around of `pg_attyp == TIMESTAMPOID && preferred_affinity == SQLITE_INTEGER` to `TODO` comment in https://github.com/pgspider/sqlite_fdw/pull/95/commits/bd75d245bdd74b649805cad193cf3375af828dfd , please verify. Also please notice UPDATE with `timestamp` with `column_type` was untested and not correct and still leave untested and not correct.

Reply:
Could you please remove those code completely from this PR?
The reasons:
- We would like to keep the code in master branch clean, with only official code (and comment) that has been tested and verified. No redundant comment is allowed to merge to it.
- The original purpose of this PR is for UUID only. You also have a plan to release a separated PR for `timestamp`. Therefore, it is better to keep only code (and also comment) about `UUID` only in this PR. You can move the code for `timestamp` to your new PR.

Reply:
Thanks for clarification, @son-phamngoc ! Changed in https://github.com/pgspider/sqlite_fdw/pull/95/commits/fcb8b8aca94bd6628a524aa234cea7d7f5c05834.

Reply:
Thanks for fixing. I confirmed. Please help me to resolve this comment.